.github/workflows/ubuntu.yml

```diff
@@ -32,10 +32,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v3
-    - name: Set up Python 3.8
+    - name: Set up Python 3.9
```

Comment:
Is this a requirement of the new versions of TF/Keras?

Reply:
yes, tensorflow and keras are not compatible with python < 3.9